music/src/core/player.ts

```diff
@@ -201,20 +224,37 @@ export class Player {
     }
     return this.synths.get(instrument);
   }
+}
 
-  /**
-   * Stop playing the currently playing sequence right away.
-   */
-  stop() {
-    if (this.currentPart) {
-      this.currentPart.stop();
-      this.currentPart = null;
-    }
-    Tone.Transport.clear(this.scheduledStop);
-    this.scheduledStop = null;
+/**
+ * A `NoteSequence` player based on Tone.js that uses SoundFont samples. The
+ * `loadSamples` method must be called before `start` so that the necessary
```

Comment:
`loadSamples` is no longer *required* before `start`.  Can you also avoid having to explicitly call initialize (see below)?

I think you should expand this a bit more, maybe even including a bit of example usage. There are few things to note with regards to when there will be a delay.  For example:

> When you start playing a sequence, the SoundFont information and required samples must be downloaded, which will lead to a delay.
> To avoid this delay when starting playback, you can pre-initialize the class and load the required samples, e.g.:
> ```js
> player.initialize().then(() => player.loadSamples(seq));  
> ```
> Once this completes `player.start(seq)` will start without delay.

Also, if you lazily call initialize in `loadSamples`, `player.loadSamples(seq)` will be sufficient.

Reply:
Fixed comments.  Also, SoundFontPlayer no longer has an initialize method.